app/views/paye/PayeTaxFreeAmountView.scala.html

```diff
@@ -28,7 +28,10 @@
 
 @(viewModel:PayeTaxFreeAmount)(implicit messages: Messages, request: Request[_], formPartialRetriever: FormPartialRetriever, appConfig: ApplicationConfig)
 
-@payeMainView(title = Messages("paye.ats.tax_free_amount.title") + Messages("generic.to_from", viewModel.taxYearStart, viewModel.taxYearEnd)) {
+@payeMainView(
+	title = Messages("paye.ats.tax_free_amount.title") + Messages("generic.to_from", viewModel.taxYearStart, viewModel.taxYearEnd),
+	mainViewParameters = MainViewParameters(true, true)
```

Comment:
You should be able to remove these from each file once you have a default implementation of MainViewParameters created in the main views